### PR TITLE
Update community.md

### PR DESCRIPTION
**Pull request by InessaPawson**
Expanding on the version written by @rgommers .

### CODE REVIEW

File: content/en/community.md [hunk 1]
```diff
@@ -3,28 +3,77 @@ title: Community
...
 
-NumPy offers a number of communication channels available to individuals interested in using and contributing to the project.
 
-We welcome contributions and contributors of all kinds - whether they come as contributions to code, participation in the community, opening issues and pointing out bugs, or simply sharing your work with others.
+# Community
 
-### Join the NumPy community
+NumPy is an open source project developed by a very diverse group of [contributors](https://numpy.org/team.md).
 
-If you’re interested in joining the NumPy community (yay!) we recommend checking out the [NumPy Contributing guide](https://www.numpy.org/devdocs/dev/index.html). This contains information about ways to get involved, the skills that are useful for each project area or activity, how to get started working on code, documentation or the website, and other ways that you can become a part of the NumPy community.
```
Comment (rgommers) [inline content/en/community.md]: I liked this sentence, I prefer to keep it

File: content/en/community.md [hunk 2]
```diff
@@ -3,28 +3,77 @@ title: Community
 sidebar: false
 ---
 
-NumPy offers a number of communication channels available to individuals interested in using and contributing to the project.
 
-We welcome contributions and contributors of all kinds - whether they come as contributions to code, participation in the community, opening issues and pointing out bugs, or simply sharing your work with others.
+# Community
```
Comment (rgommers) [inline content/en/community.md]: Note: this isn't needed, it's already given above in the `title:` metadata. Also, `<h1>` section headers don't work, start from h2 (`##`).

File: content/en/community.md [hunk 3]
```diff
@@ -3,28 +3,77 @@ title: Community
...
+We offer several communication channels to learn, share your knowledge and connect with others within the NumPy community.
 
-### Participate online
+The NumPy leadership has made a strong commitment to creating an open, inclusive, and positive community. Please read the [NumPy Code of Conduct](https://www.numpy.org/code-of-conduct.md) for guidance on how to interact with others in a way that makes the community thrive.
 
-NumPy also has a number of online communication channels to help keep in touch.
+## Online Communities and Social Resources
 
-As you interact with others in the NumPy community, remember that we have a strong commitment to being an open, inclusive, and positive community. Please read the [NumPy Code of Conduct](https://www.numpy.org/devdocs/dev/conduct/code_of_conduct.html) for guidance on how to interact with others in a way that makes the community thrive.
+### [NumPy mailing list](https://mail.python.org/mailman/listinfo/numpy-discussion)
```
Comment (rgommers) [inline content/en/community.md]: The rendering as section headers looks a little odd. Content of the sections makes sense, however I'd rather use links in a more obvious way as part of a sentence or start of a list item.

File: content/en/community.md [hunk 4]
```diff
@@ -3,28 +3,77 @@ title: Community
...
+### [Gitter](https://gitter.im/numpy/numpy)
+
+ a real-time chat room where users and community members help each other.
+
+
+### [Slack](https://app.slack.com/client/TM52KRJ2K/CLRQESU4T)
+
+a real-time chat room to ask questions about contributing to NumPy.
+
+If you’ve found a security vulnerability in NumPy, please report it using the procedure documented at[ https://tidelift.com/docs/security](https://tidelift.com/docs/security)
```
Comment (rgommers) [inline content/en/community.md]: This is missing a section header now.

File: content/en/community.md [hunk 5]
```diff
@@ -3,28 +3,77 @@ title: Community
...
+
+
+## Study Groups and Meetups
+
+[PyData](https://www.meetup.com/pro/pydata/)
```
Comment (rgommers) [inline content/en/community.md]: I like this. Will add a sentence to it to explain a little more.

File: content/en/community.md [hunk 6]
```diff
@@ -3,28 +3,77 @@ title: Community
...
+
+
+## Conferences and Events
```
Comment (rgommers) [inline content/en/community.md]: Also useful. I'd add the other SciPy conferences, and a note on sprints and where they get announced.

File: content/en/community.md (hunk 1 above)
Comment (rgommers) [reply content/en/community.md]: The first part is kept at the bottom of the page, the second part is discarded.

Comment (rgommers) [reply content/en/community.md]: decided to leave as is anyway

File: content/en/community.md (hunk 3 above)
Comment (rgommers) [reply content/en/community.md]: I didn't find a clearly better way, without going to a button solution like https://jupyter.org/community

**Comment by rgommers** [comment]
Okay, updated. This looks okay I think.

**Comment by rgommers** [comment]
I'll merge it, we can iterate a little if needed, but this is pretty close. Thanks @InessaPawson

File: content/en/community.md (hunk 3 above)
Comment (InessaPawson) [reply content/en/community.md]: @rgommers The primary purpose of my PR was to upload the content. Good job on improving the formatting! 
Not sure about the buttons. What do you think about placing active icons before each header?

Comment (rgommers) [reply content/en/community.md]: > What do you think about placing active icons before each header?

Something like that could be nice. I'd like to postpone such changes till after we get the first version of the site up though - visual improvements like this are time consuming and aren't blockers.

**Pull request merged**